src/components/inputs/FormattedNumberInputNew.tsx

```diff
@@ -0,0 +1,116 @@
+import { InputNumber } from 'antd'
+import { CSSProperties } from 'react'
+import { formattedNum } from 'utils/formatNumber'
+
+import { FormItemExt } from 'components/formItems/formItemExt'
+
+export default function FormattedNumberInputNew({
```

Comment:
This is a transitory function, just used to make bridging the changes (without having broken code)

Reply:
This is eventually renamed to `FormattedNumberInput` once the old one is finally discarded in the final PR